akka-remote/src/main/resources/reference.conf

```diff
@@ -1090,6 +1090,10 @@ akka {
           # Only used when transport is aeron-udp.
           client-liveness-timeout = 20 seconds
 
+          # Timout after after which an uncommitted publication will be unblocked
+          # Only used when transport is aeron-udp.
+          publication-unblock-timeout = 40 seconds
```

Comment:
How did you find that this was needed? Default is 10 seconds? Why do we need a different default?

This is a property for the (embedded) media driver, and unless we have strong reasons for adding our own (different) configuration for those I think we should use defaults and users can override with `-D` System property.

Reply:
> Every test failed :) reason is in the commit msg
>
> The media driver won’t start with the default as we already override client
liveness timeout and this has to be bigger than that. I picked 40s as it
keeps the same ratio as the defaults

ok, good

Reply:
I am wondering the same, but I see it in the commit.